include/pagmo/problem.hpp

```diff
@@ -1058,6 +1139,7 @@ class problem
      * - the <tt>%gradient_sparsity()</tt> and <tt>%hessians_sparsity()</tt> methods of the UDP fail basic sanity checks
      *   (e.g., they return vectors with repeated indices, they contain indices exceeding the problem's dimensions,
      *   etc.).
+     * - the integer part of the problem is larger than the problem size.
```

Comment:
I think we are missing some failure modes for ``check_problem_bounds()``: the integer bounds are not integers and the infinity check (if added above).